app/src/main/res/layout-land/activity_main.xml

```diff
@@ -0,0 +1,182 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/activity_main"
+    style="@style/ConstraintStyleLandscape"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    tools:context=".activity.CalculatorActivity">
+
+    <TextView
+        android:id="@+id/text_result"
+        style="@style/TextViewStyle"
+        android:layout_width="match_parent"
+        android:layout_height="@dimen/height_textview_landscape_xbig"
```

Comment:
why do you hardcode the height and the width? you should use the constraints to avoid hardcode sizes

Reply:
Used constraints to avoid hardcode sizes